changelog.md

```diff
@@ -299,6 +299,8 @@
 
 - Added `copyWithin` [for `seq` and `array` for JavaScript targets](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/copyWithin).
 
+- Added `initialSize` to `parsecsv.open` for the initial size to allocate for the rows of the CSV.
```

Comment:
> Add initialSize for parsecsv.open, inspired by Table.initTable( initialSize ), tiny diff, no API break.

but `Table.initTable` was to avoid users having to initialize by hand, whereas in the case of parsescv, IIUC, users don't need to initialize by hand, so I'm not sure this micro-optimization is needed, unless you can show a benchmark where it makes a difference. The other costs invovled (io) likely dwarf whatever savings coming from pre-allocation of buffer capacity (and 32 seems arbitrary)

(also, 0 should always be an option, no point in restricting this with `Positive`)

Reply:
`32` is arbitrary, I just copied what `Table` uses.
It can do string streams too, so not much IO there.

Reply:
> It can do string streams too, so not much IO there.

good point. But other points remain; can you show a benchmark where this makes a difference?

Reply:
This was not about benchmarks, this is about correctness of the code,
what if I know that I will be always working with CSV of exactly 64 rows,
why I can not just allocate it from start like you can do in C++ ?.

Reply:
that's still about performance, not correctness though (the code is still correct and will parse the CSV correctly). I'm not opposed to it if you can show a reasonable benchmark showing the need for it